.circleci/config.yml

```diff
@@ -118,16 +128,10 @@ workflows:
                 - main
     jobs:
       - build
-      - quality/check-quality-job:
-          custom_rakefile: rakelib/quality.rake
-          image_tag: *quality_gem_version
+      - quality
   quality:
-    when:
-      equal: [<< pipeline.git.branch >>, "main"]
     jobs:
-      - quality/check-quality-job:
-          custom_rakefile: rakelib/quality.rake
-          image_tag: *quality_gem_version
+      - quality
   build:
     jobs:
```

Comment:
[punchlist] Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker